### PR TITLE
Adding image of Dryville for Issue #16

### PR DESCRIPTION
**Pull request by galengorski**
Not sure if I added the image to the .gitignore file correctly for issue #16 . If not, would you mind helping me out. Thanks!

### CODE REVIEW

**Comment by jds485** [comment]
Just to confirm, did you download the .jpg to your directory? What do you see after running `git status`? If you could paste the output in a reply that would be helpful. I'm curious if what you entered in the `.gitignore` works because it's recognizing the source of the file

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+<img src="https://prd-wret.s3.us-west-2.amazonaws.com/assets/palladium/production/s3fs-public/styles/carousel_large/public/thumbnails/image/wss-wu-dryville.jpg?itok=uxY0Fwj1"/>
```
Comment (jds485) [inline .gitignore]: This should just be the filepath. In this case, `wss-wu-dryville.jpg`

Comment (galengorski) [reply .gitignore]: Oh I see, I misunderstood the directions, that's my fault. I should be able to quickly redo it, would it be best to open a new PR in that case?

Comment (jds485) [reply .gitignore]: You can continue adding to this PR! Adding another commit to the branch will also add the commit to this PR

**Review by jds485 — APPROVED** [review]
Looks good!

**Pull request merged**